python/cugraph/cugraph/structure/property_graph.py

```diff
@@ -1127,7 +1364,29 @@ def annotate_dataframe(self, df, G, edge_vertex_col_names):
 
         Examples
         --------
-        >>>
+        >>> import cugraph
+        >>> import cudf
+        >>> from cugraph.experimental import PropertyGraph
+        >>> df = cudf.DataFrame(columns=["src", "dst", "some_property"],
+        ...                              data=[(99, 22, "a"),
+        ...                                    (98, 34, "b"),
+        ...                                    (97, 56, "c"),
+        ...                                    (96, 88, "d"),
+        ...                                  ])
+        >>> pG = PropertyGraph()
+        >>> pG.add_edge_data(df, type_name="etype", vertex_col_names=("src", "dst"))
+        >>> G = pG.extract_subgraph(create_using=cugraph.Graph(directed=True))
+        >>> algo_result = cudf.DataFrame({"from":df.src,
```

Comment:
I think is is an acceptable variable name that indicates to users that this API is often used for algo results, but you might add a comment in the example saying that this dataframe represents a typical DF returned from an algo run, since it otherwise seems odd to name a dataframe you're creating "algo_result".

Or even better, can you just run an actual algo like `pagerank` or something that returns a dataframe?

Reply:
added a comment to the docstring